examples/powertools-examples-core/README.md

```diff
@@ -5,17 +5,17 @@ This project demonstrates the Lambda for Powertools Java module - including
 [tracing](https://docs.powertools.aws.dev/lambda/java/core/tracing/), and
 [metrics](https://docs.powertools.aws.dev/lambda/java/core/metrics/).
 
-It is made up of the following:
+We provide examples for the following tools:
+* [AWS SAM](sam/) 
+* [AWS CDK](cdk/)
 
-- [App.java](src/main/java/helloworld/App.java) - Code for the application's Lambda function.
-- [events](events) - Invocation events that you can use to invoke the function.
-- [AppTests.java](src/test/java/helloworld/AppTest.java) - Unit tests for the application code. 
-- [template.yaml](template.yaml) - A template that defines the application's AWS resources.
+For all the tools, the example application is the same, and consists of the following files:
 
-## Deploy the sample application
+- [App.java](sam/src/main/java/helloworld/App.java) - Code for the application's Lambda function.
```

Comment:
I thought the code was going to end up in a shared common dir, rather than the subdir of one of the CI tools - was this not practical?

Reply:
That discussion was resolved, it seems, but I agreed with you that duplicating the sample app for now is a better solution, since SAM doesn't work well otherwise